sympy/polys/tests/test_numberfields.py

```diff
@@ -637,9 +637,9 @@ def test_AlgebraicNumber():
 
     a = AlgebraicNumber(sqrt(2))
     b = to_number_field(sqrt(2))
-    assert a.args == b.args == (sqrt(2), Tuple())
+    assert a.args == b.args == (sqrt(2), Tuple(1, 0))
```

Comment:
I agree. ``None`` is what I would have expected as the default. But I did not want to make too many changes because of a simple printing bug. Perhaps that could be a separate PR. Or do you think it would be reasonable to add another commit to this PR?
Even the change to ``Tuple(1, 0)`` was not necessary, but I felt it should made in case someone would want to use the ``scoeffs`` set for something more useful than just a trivial test.

Anyway, there is a lot to do in AlgebraicField. Maybe the final form of AlgebraicNumber can be fixed only after that is done.